subscribie/blueprints/admin/__init__.py

```diff
@@ -682,6 +682,36 @@ def is_active_gocardless(email):
     return False
   return dict(is_active_gocardless=is_active_gocardless)
 
+
+def is_empty(string):
+    return False if string and str(string).strip() else True
+
+
+def is_subscription_active(gocardless_subscription_id):
```

Comment:
This is good, there are multiple possible subscription statuses which may be returned: https://github.com/Subscribie/subscribie/issues/146#issuecomment-621411861 

please change to `get_subscription_status(gocardless_subscription_id)`
Which returns a string of the status from the api e.g 

```
...
return response.status
...
```

In the UI, simply return the status value from the api response

Reply:
I guess other payment processors also have specific subscription statuses so I was thinking to simplify them by only recognizing active and inactive.
Alright, will return original status.